### PR TITLE
Feature#1/UI designs app screens

### LINKED ISSUE #1: Create UI Design for App screens
# REQUIREMENTS

- We are going to do the following  UI SCREENS for App Screens.

image
image
image

# Acceptance Criteria
List all required acceptance criteria
- [ ] Nice UI
- [ ] Good folders, files and code structure.

### PR DESCRIPTION
**Pull request by ChantalRamirez**
# Description
UI design of all screen
 

https://user-images.githubusercontent.com/58367346/111856805-e297e280-88f2-11eb-8239-848ae881b6c0.mp4

### CODE REVIEW

**Review by vmagana3 — DISMISSED** [review]
Good work!!

**Review by GracielaJazmin — DISMISSED** [review]

File: AlertCovidApp/src/components/atoms/GeneralApp/Button.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {View, Text, TouchableOpacity} from 'react-native';
+import {styles} from '../../../styles/Home';
+import {colors} from '../../../utils/Colors'
+
+const Button = ({navigation, RouteGo, textButton, colorBackground}) => {
+  return (
+    <TouchableOpacity 
+    style={[styles.ButtonWellBad, {backgroundColor: textButton === 'WELL' ? colors.green : colors.orange},]}
```
Comment (fernandoamz) [inline AlertCovidApp/src/components/atoms/GeneralApp/Button.js]: Try to do this line easier to read

File: AlertCovidApp/src/components/atoms/GeneralApp/ButtonLarge.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import {View, Text, TouchableOpacity} from 'react-native';
+import Icons from 'react-native-vector-icons/Feather';
+import {styles} from '../../../styles/Home';
+import {colors} from '../../../utils/Colors'
+
+const ButtonLarge = ({navigation, RouteGo, textButton}) => {
+  return (
+    <TouchableOpacity 
+    style={[styles.ButtonLarge, {backgroundColor: textButton === 'TEST RESULTS' ? colors.blue : colors.gray},]} 
+    onPress={ () => navigation.navigate(RouteGo)
```
Comment (fernandoamz) [inline AlertCovidApp/src/components/atoms/GeneralApp/ButtonLarge.js]: Review the identation

File: AlertCovidApp/src/components/atoms/GeneralApp/Form.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { View, Text, TouchableOpacity } from 'react-native';
+import {styles} from '../../../styles/HistoryTest/index';
+import IconTabMenu from 'react-native-vector-icons/Feather';
+
+const FormLine = ({Label, TextInput}) => {
+    return (
+      <View style={{display: 'flex', flexDirection: 'row',justifyContent:'space-between',  marginLeft: 20, marginBottom:15, marginTop:2}}>
+        <Text style={{color:'#888888', marginRight: 50}}>{Label}</Text>
+        <Text style={{color:'#888888'}}>{TextInput}</Text>
```
Comment (fernandoamz) [inline AlertCovidApp/src/components/atoms/GeneralApp/Form.js]: All the colors should be in a constants file

File: AlertCovidApp/src/components/atoms/GeneralApp/ScrollWhite.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            display: 'flex',
+            flexDirection: 'row',
+            justifyContent: 'flex-start',
+            marginBottom: 5,
+            marginTop: 5,
+        }}>
+        <Text>Data Reported</Text>
+      </View>
+
+        <FormLine Label={'Date'} TextInput={'10/03/2021'}></FormLine>
```
Comment (fernandoamz) [inline AlertCovidApp/src/components/atoms/GeneralApp/ScrollWhite.js]: Why this value is hardcoded?

File: AlertCovidApp/src/components/atoms/GeneralApp/ScrollWhite.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+          marginTop: 5,
+        }}>
+        <Text>Test Result</Text>
+      </View>
+
+      <FormLine Label={'Date of rest'} TextInput={'06/03/2021'}></FormLine>
+      <FormLine Label={'Date Result'} TextInput={'09/03/2021'}></FormLine>
+      <FormLine Label={'Test type'} TextInput={'Quick test'}></FormLine>
+      <FormLine Label={'Test result'} TextInput={'Test failed'}></FormLine>
+      <FormLine Label={'Symptoms on the day of test'} TextInput={'Yes'}></FormLine>
```
Comment (fernandoamz) [inline AlertCovidApp/src/components/atoms/GeneralApp/ScrollWhite.js]: The same case

File: AlertCovidApp/src/components/atoms/HistoryTest/History.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+        <TouchableOpacity>
+          <Text style={{ fontSize: 10,color: colors.gray}}>Not Clear/</Text>
+        </TouchableOpacity>
+        <TouchableOpacity>
+          <Text style={{ fontSize: 10,color: colors.gray}}>Test failed</Text>
+        </TouchableOpacity>
+      </View>
+
+      <View style={{display: 'flex', flexDirection: 'row',justifyContent:'space-between',  marginLeft: 20}}>
+        <Text style={{color:'#888888'}}>Viral Test</Text>
```
Comment (fernandoamz) [inline AlertCovidApp/src/components/atoms/HistoryTest/History.js]: Text and styles should be in a constant file. The styles should be inside styles tag

File: AlertCovidApp/src/components/atoms/HistoryTest/History.js [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+    </View>
+
+    <View style={{borderBottomWidth: 1, marginTop:15, marginBottom:10, marginHorizontal:10 }}>
+      <View
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          justifyContent: 'space-around',
+          marginBottom: 5,
+          marginTop: 5,
```
Comment (fernandoamz) [inline AlertCovidApp/src/components/atoms/HistoryTest/History.js]: same case

File: AlertCovidApp/src/components/atoms/HistoryTest/History2.js [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+      </View>
+    </View>
+  </ScrollView>
+
+
+  );
+};
+
+export default HistoryData;
```
Comment (fernandoamz) [inline AlertCovidApp/src/components/atoms/HistoryTest/History2.js]: New line here

File: AlertCovidApp/src/components/atoms/Home/BackgroundImage.js [hunk 9]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {ImageBackground} from 'react-native';
+import { styles } from '../../../styles';
+
+const BackgroundImage = () => {
+  return (
+     <ImageBackground style={styles.ImageBackground} source={require('../../../assets/images/Welcome.png')}></ImageBackground>
```
Comment (fernandoamz) [inline AlertCovidApp/src/components/atoms/Home/BackgroundImage.js]: import the image route

File: AlertCovidApp/src/components/atoms/Symptoms/MultiSelect.js [hunk 10]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+
+
+
```
Comment (fernandoamz) [inline AlertCovidApp/src/components/atoms/Symptoms/MultiSelect.js]: only should be one line

File: AlertCovidApp/src/navigations/AppNavigation.js [hunk 11]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+// import { NavigationContainer } from '@react-navigation/native';
```
Comment (fernandoamz) [inline AlertCovidApp/src/navigations/AppNavigation.js]: can you remove this line?

File: AlertCovidApp/src/screens/ScanQr.js [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+          </TouchableOpacity>
+        </View>
+        </View>
+        <TapMenu navigation ={navigation} IconSelected={'History'}></TapMenu>
+
+      </View>
+    )
+}
+
+export default TestResults;
```
Comment (fernandoamz) [inline AlertCovidApp/src/screens/ScanQr.js]: new line

**Review by fernandoamz — CHANGES_REQUESTED** [review]
You need add a blank line at the end of every file

File: AlertCovidApp/src/components/atoms/GeneralApp/ScrollWhite.js (hunk 4 above)
Comment (JacintoOrtiz15745) [reply AlertCovidApp/src/components/atoms/GeneralApp/ScrollWhite.js]: What's mean hardcoded?

File: AlertCovidApp/src/components/atoms/GeneralApp/Form.js [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+          style={styles.menuIcon}
+        />
+       </TouchableOpacity>
+
+      </View>
+    
+    )
+}
+
+export default FormLine;
```
Comment (LeyaimMagma) [inline AlertCovidApp/src/components/atoms/GeneralApp/Form.js]: add blank space

File: AlertCovidApp/src/components/atoms/GeneralApp/ScrollWhite.js [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+    <View style={styles.ContainerDataResults}>
+        <View style={{borderBottomWidth: 1, marginTop:15, marginBottom:10, marginHorizontal:10}}>
+        <View
+            style={{
+            display: 'flex',
+            flexDirection: 'row',
+            justifyContent: 'flex-start',
+            marginBottom: 5,
+            marginTop: 5,
+        }}>
```
Comment (LeyaimMagma) [inline AlertCovidApp/src/components/atoms/GeneralApp/ScrollWhite.js]: move to stylessheet

File: AlertCovidApp/src/components/atoms/GeneralApp/ScrollWhite.js [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+      <FormLine Label={'Test type'} TextInput={'Quick test'}></FormLine>
+      <FormLine Label={'Test result'} TextInput={'Test failed'}></FormLine>
+      <FormLine Label={'Symptoms on the day of test'} TextInput={'Yes'}></FormLine>
+      
+    </View>
+  </View>
+    )
+}
+
+export default ScrollWhite;
```
Comment (LeyaimMagma) [inline AlertCovidApp/src/components/atoms/GeneralApp/ScrollWhite.js]: always end the files with a blank space

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

File: AlertCovidApp/src/components/atoms/BackgroundImage.js [hunk 16]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {View, Text, ImageBackground} from 'react-native';
+import {styles} from '../../styles';
+
+const BackgroundImage = () => {
+  return (
+    <View>
+      <ImageBackground
+        style={styles.ImageBackground}
+        source={require('../../assets/images/Welcome.png')}></ImageBackground>
```
Comment (fernandoamz) [inline AlertCovidApp/src/components/atoms/BackgroundImage.js]: Add the import for this image.

File: AlertCovidApp/src/components/atoms/GeneralApp/Button.js [hunk 17]
```diff
@@ -0,0 +1,19 @@
...
+import {View, Text, TouchableOpacity} from 'react-native';
+import {styles} from '../../../styles/Home';
+import {colors} from '../../../utils/Colors';
+
+const Button = ({navigation, RouteGo, textButton, colorBackground}) => {
+  return (
+    <TouchableOpacity  
+      style={[
+        styles.ButtonWellBad,
+        {backgroundColor: textButton === 'WELL' ? colors.green : colors.orange},
```
Comment (fernandoamz) [inline AlertCovidApp/src/components/atoms/GeneralApp/Button.js]: make this line easier to read

File: AlertCovidApp/src/components/atoms/HistoryTest/History.js [hunk 18]
```diff
@@ -0,0 +1,58 @@
...
+    <ScrollView style={styles.ContainerDataUser}>
+      <View style={styles.SubContainerDataUser}>
+        <View style={styles.SubContainerTwoDataUser}>
+          <Text>Sunday</Text>
+          <Text>28/03/2021</Text>
+          <TouchableOpacity>
+            <Text style={{fontSize: 10, color: colors.gray}}>Not Clear/</Text>
+          </TouchableOpacity>
+          <TouchableOpacity>
+            <Text style={{fontSize: 10, color: colors.gray}}>Test failed</Text>
```
Comment (fernandoamz) [inline AlertCovidApp/src/components/atoms/HistoryTest/History.js]: All texts should be in a constants file

File: AlertCovidApp/src/components/atoms/Profile/UserDataScroll.js [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+          style={styles.ContainerPassword}>
+          <Text>PASSWORD</Text>
+          <TouchableOpacity>
+            <Text>Change</Text>
+          </TouchableOpacity>
+        </View>
+        <View style={styles.Line}>
+          <Text style={styles.Text}>* * * * * * * * * *</Text>
+        </View>
+      </View>
```
Comment (fernandoamz) [inline AlertCovidApp/src/components/atoms/Profile/UserDataScroll.js]: Did you try using a TextInput?

File: AlertCovidApp/src/screens/HistorySymptoms.js [hunk 20]
```diff
@@ -0,0 +1,42 @@
...
+          <Text>Symptoms</Text>
+        </TouchableOpacity>
+        <TouchableOpacity
+          style={styles.ContainerTest}
+          onPress={() => navigation.navigate('HistoryTest')}>
+          <Text>Test</Text>
+        </TouchableOpacity>
+      </View>
+
+      <History2 navigation={navigation} />
```
Comment (fernandoamz) [inline AlertCovidApp/src/screens/HistorySymptoms.js]: Change the name of this variable, Review the good practices to name a variable

**Review by vmagana3 — DISMISSED** [review]
Good work!!

**Review by fernandoamz — CHANGES_REQUESTED** [review]

File: AlertCovidApp/src/components/atoms/BackgroundImage.js (hunk 16 above)
Comment (ChantalRamirez) [reply AlertCovidApp/src/components/atoms/BackgroundImage.js]: ok

File: AlertCovidApp/src/components/atoms/GeneralApp/Button.js (hunk 17 above)
Comment (ChantalRamirez) [reply AlertCovidApp/src/components/atoms/GeneralApp/Button.js]: What do you mean "Make easier" ?? sorry

Comment (ChantalRamirez) [reply AlertCovidApp/src/components/atoms/GeneralApp/Button.js]: We use prettier, but we don´t  understand

File: AlertCovidApp/src/utils/Constants/History.js [hunk 21]
```diff
@@ -0,0 +1,12 @@
...
+    Symptoms: 'Symptoms',
+    Test: 'Test',
+    sunday:'Sunday',
+    date: '28/03/2021',
+    notClear:'Not Clear/',
+    testFailed:'Test failed',
+    viral: 'Viral Test',
+    negative:'Negative',
+    antibody:'Antibody Test'
+};
```
Comment (LeyaimMagma) [inline AlertCovidApp/src/utils/Constants/History.js]: add blank space at the end

File: AlertCovidApp/src/utils/Constants/HistoryConstants.js [hunk 22]
```diff
@@ -0,0 +1,5 @@
+export const HistoryConstants = {
+    History: 'History',
+    Symptoms: 'Symptoms',
+    Test: 'Test',
+};
```
Comment (LeyaimMagma) [inline AlertCovidApp/src/utils/Constants/HistoryConstants.js]: add blank space at the end

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

**Review by fernandoamz — DISMISSED** [review]

**Review by Joaquin-Niembro — APPROVED** [review]

**Review by jjcr11 — APPROVED** [review]

**Review by fernandoamz — APPROVED** [review]

**Pull request merged**